src/examples/local_test.yml

```diff
@@ -3,10 +3,11 @@ description: >
 usage:
   version: 2.1
   orbs:
-    sam: circleci/aws-sam-serverless@x.y.z
+    sam: circleci/aws-sam-serverless@3.0
   jobs:
     build_and_package:
-      executor: sam/default # Or your own. Machine executor recommended.
+      machine:
+        image: ubuntu-2004:202010-01
```

Comment:
Same comment here, though we wouldn't we used the bundled executor in the example?

Reply:
I removed the executor from the orb. It might be shorter in the example but wrapping existing circleci default config I figured is probably more confusing in general.